Casks/inkstitch.rb

```diff
@@ -30,7 +30,9 @@
   end
 
   preflight do
-    system_command "/bin/mkdir", args: ["-p", "#{ENV["HOME"]}/Library/Application Support/org.inkscape.Inkscape/config/inkscape/extensions"]
+    system_command "/bin/mkdir",
+                   args: ["-p", "#{ENV["HOME"]}/Library/Application Support \
+                   /org.inkscape.Inkscape/config/inkscape/extensions"]
```

Comment:
```suggestion
                   args: ["-p", "#{ENV["HOME"]}/Library/Application Support\
                                 /org.inkscape.Inkscape/config/inkscape/extensions"]
```
You don't want that last space, I think. Might be nicer to align the string continuation too.

Reply:
Thanks @carlocab :)